ksqldb-rest-app/src/test/java/io/confluent/ksql/rest/integration/PullQueryFunctionalTest.java

```diff
@@ -129,6 +129,7 @@
       .withProperty(KsqlRestConfig.ADVERTISED_LISTENER_CONFIG, "http://localhost:8188")
       .withProperty(KsqlConfig.KSQL_QUERY_PULL_ENABLE_STANDBY_READS, true)
       .withProperty(KsqlConfig.KSQL_STREAMS_PREFIX + "num.standby.replicas", 1)
+      .withProperty(KsqlConfig.KSQL_STREAMS_PREFIX + "cache.max.bytes.buffering", 10000)
```

Comment:
Why did you add these?

Reply:
I just added them to one of the pull query tests because I wanted to exercise the cache bypassing code in a functional test.  It's hard to set up a realistic scenario in unit tests since it's not easy to construct all of the state store layers as they actually exist.

If something basic goes wrong in the reflection or types with underlying streams when faced with the cache, these tests should fail.  The real test for this will be the automated benchmarks which test that this bypassing effect works as intended.

Reply:
Ok, makes sense